aspnet/migration/rc1-to-rc2.rst

```diff
@@ -0,0 +1,352 @@
+IWebApplicationBuilder         IWebHostBuilder
+WebApplicationBuilder          WebHostBuilder
+IWebApplication                IWebHost
+WebApplication                 WebHost
+WebApplicationOptions          WebHostOptions
+WebApplicationDefaults         WebHostDefaults
+WebApplicationService          WebHostService
+WebApplicationConfiguration    WebHostConfiguration
+===========================    =========================
+
+The ``commands`` section of ``project.json`` has been removed completely. Use ``run`` or ``dotnet <dllname>`` instead.  
+
+
+IHostingEnvironment changes 
+---------------------------
+
+All environment variables are now prefixed with the ``ASPNETCORE_`` prefix.
+
+======================  =========================    
+Old prefix              New prefix                           
+======================  =========================  
+ASPNET_WEBROOT          ASPNETCORE_WEBROOT
+ASPNET_SERVER           ASPNETCORE_SERVER
+ASPNET_APP              ASPNETCORE_APP
+ASPNET_ENVIRONMENT      ASPNETCORE_ENVIRONMENT
+ASPNET_DETAILEDERRORS   ASPNETCORE_DETAILEDERRORS
+======================  =========================  
+
+ASP.NET 5 was renamed to ASP.NET Core 1.0. Also, MVC and Identity are now part of ASP.NET Core. ASP.NET MVC 6 is now ASP.NET Core MVC. ASP.NET Identity 3 is now ASP.NET Core Identity.
+
+The hosting configuration keys are now consistent with the command line, environment variables, and ``hosting.json`` values. The ``Microsoft.AspNet.Hosting.json`` configuration file was renamed to ``hosting.json``.
+
+``hosting.json`` is no longer read by default, users need to explicitly add their own configuration sources and pass a configuration to ``WebHostBuilder``. The default server URL and port are ``localhost:5000``.
+
+
+Runtime Services Changes
+------------------------
+
+You must modify code in the ``Startup`` class that uses ``IApplicationEnvironment`` to use ``IHostingEnvironment``:
+
+Change:
+
+.. code-block:: c# 
+
+  public Startup(IApplicationEnvironment applicationEnvironment)
+  {
+     var builder = new ConfigurationBuilder()
+       .SetBasePath(applicationEnvironment.ApplicationBasePath)
+  }
+
+To: 
+
+.. code-block:: c#
+
+  public Startup(IHostingEnvironment hostingEnvironment)
+  {
+     var builder = new ConfigurationBuilder()
+      .SetBasePath(hostingEnvironment.ContentRootPath)
+  }
+
+  
+Namespace and package id changes
+---------------------------------- 
+
+All Microsoft.AspNet.\* namespaces are renamed to Microsoft.AspNetCore.\*. 
+The EntityFramework.\* packages and namespaces are changing to Microsoft.EntityFrameworkCore.\*.
+All ASP.NET Core package versions are now 1.0.0-\*.
+
+Working with IIS
+----------------
+
+HttpPlatformModule
+^^^^^^^^^^^^^^^^^^
+
+HttpPlatformModule was replaced by ASP.NET Core Module. The web.config created by the Publish IIS tool now configures IIS to use ASP.NET Core Module instead of HttpPlatformHandler to reverse-proxy requests to Kestrel.
```

Comment:
Note that the package was also renamed: `Microsoft.AspNetCore.IISPlatformHandler` -> `Microsoft.AspNetCore.Server.IISIntegration`